Automattic/wp-calypso
# Domain management: add connect to an existing site as a secondary action

Related to https://github.com/Automattic/wp-calypso/pull/80998#issuecomment-1698692073.

## Proposed Changes

This PR adds the "Connect to an existing site" action within the table row actions:

![image](https://github.com/Automattic/wp-calypso/assets/26530524/6f5270c5-0e41-4d87-8849-8379ff5e53b4)

## Testing Instructions

Find a domain-only entry and verify that:

1. Clicking "Add site" starts the site creation flow
2. Clicking the three dots and clicking the "Connect to an existing site" menu item starts the site transfer flow

## Pre-merge Checklist



~- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?~
~- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?~
